Tensile/Source/CMakeLists.txt

```diff
@@ -104,7 +104,7 @@ else()
     set_property( CACHE Tensile_CODE_OBJECT_VERSION PROPERTY STRINGS V2 V3)
     set_property( CACHE Tensile_COMPILER PROPERTY STRINGS hipcc)
     set(Tensile_ARCHITECTURE all CACHE STRING "Which GPU architecture to use")
-    set_property( CACHE Tensile_ARCHITECTURE PROPERTY STRINGS all gfx000 gfx900 gfx906:xnack- gfx908:xnack- gfx90a:xnack- gfx1010 gfx1011 gfx1012 gfx1030)
+    set_property( CACHE Tensile_ARCHITECTURE PROPERTY STRINGS all gfx000 gfx900 gfx906:xnack+ gfx906:xnack- gfx908:xnack+ gfx908:xnack- gfx90a:xnack+ gfx90a:xnack- gfx1010 gfx1011 gfx1012 gfx1030)
```

Comment:
Okay is that from rocBLAS build?  We have our own "all" which gets passed down, just from the changed lines here I see it added in the "all" list, but note is cached variable.

Reply:
The change shouldn't be needed, even though it doesn't change the meaning of "all".  Removed.  Thanks.